test/api/assignments_test.rb

```diff
@@ -45,11 +46,8 @@ def test_api_accepts_submission_attempt
   def test_api_accepts_submission_attempt_with_multi_file_solution
     Notify.stub(:everyone, nil) do
       Xapi.stub(:exists?, true) do
-        solution = {
-          'ruby/one/file1.rb' => 'code 1',
-          'ruby/one/file2.rb' => 'code 2'
-        }
-        post '/user/assignments', {key: alice.key, solution: solution}.to_json
+        post '/user/assignments', {key: alice.key, code: 'THE CODE', path: 'ruby/one/code.rb', 'language' => 'ruby',
+          'problem' => 'one'}.to_json
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.